Makefile

```diff
@@ -82,6 +82,19 @@ test: ebpf
 bench: ebpf
 	$(call build, build -F benchmark --release, building benchmarks)
 
+coverage: RUSTFLAGS += -C instrument-coverage
+coverage: ebpf
+	$(call build, build, building retis with coverage instrumentation)
+
+merge:
+	@echo "Merging coverage data..."
+	llvm-profdata merge -sparse default_*.profraw -o retis_coverage.profdata
+
+report:
+	@echo "Generating coverage report..."
+	llvm-cov show --format=html --output-dir=coverage --instr-profile=retis_coverage.profdata target/debug/retis
```

Comment:
We can hide dependencies from showing up in the report by adding the following:

```
--ignore-filename-regex=/.cargo/registry \
--ignore-filename-regex=/rustc/
```

Also I know that a demangler can be used by adding `-Xdemangler=rustfilt`, not sure about how this improves the output.